benchmarks/bench_random_projections.py

```diff
@@ -73,8 +74,8 @@ def make_sparse_random_data(n_samples, n_features, n_nonzeros, random_state=None
         (
             rng.randn(n_nonzeros),
             (
-                rng.randint(n_samples, size=n_nonzeros),
-                rng.randint(n_features, size=n_nonzeros),
+                rng_integers(rng, n_samples, size=n_nonzeros),
```

Comment:
Same here regarding not needing to change files in the main `sklearn` files.

Reply:
Done